go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/Azure/go-autorest/autorest/to v0.2.0
 	github.com/GoogleCloudPlatform/k8s-cloud-provider v0.0.0-20190822182118-27a4ced34534
 	github.com/JeffAshton/win_pdh v0.0.0-20161109143554-76bb4ee9f0ab
-	github.com/Microsoft/go-winio v0.4.11
```

Comment:
Why the go modules update?

Reply:
the vendor (go-winio) bump is needed for this change to support GetAddressAndDialer refactor. 
xref: https://github.com/kubernetes/kubernetes/pull/83003#discussion_r327107943